nautobot/extras/tests/test_views.py

```diff
@@ -805,3 +806,71 @@ def setUpTestData(cls):
             "fallback_value": ":skull_emoji:",
             "weight": 100,
         }
+
+
+class CustomFieldTestCase(
+    ViewTestCases.BulkDeleteObjectsViewTestCase,
+    ViewTestCases.CreateObjectViewTestCase,
+    ViewTestCases.DeleteObjectViewTestCase,
+    ViewTestCases.EditObjectViewTestCase,
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.ListObjectsViewTestCase,
+):
+    model = CustomField
+
+    def test_create_object_without_permission(self):
+        # Can't have two CustomFields with the same "name"
+        for cf in CustomField.objects.all():
+            cf.delete()
+        super().test_create_object_without_permission()
+
+    def test_create_object_with_permission(self):
+        # Can't have two CustomFields with the same "name"
+        for cf in CustomField.objects.all():
+            cf.delete()
+        super().test_create_object_with_permission()
+
+    def test_create_object_with_constrained_permission(self):
+        # Can't have two CustomFields with the same "name"
+        for cf in CustomField.objects.all():
+            cf.delete()
+        super().test_create_object_with_constrained_permission()
+
```

Comment:
To help the code read better, please move these `test_*` methods to be *below* the `setUpTestData` method.